lib/ansible/plugins/action/synchronize.py

```diff
@@ -201,6 +201,7 @@ def run(self, tmp=None, task_vars=None):
 
         # Parameter name needed by the ansible module
         _tmp_args['_local_rsync_path'] = task_vars.get('ansible_rsync_path') or 'rsync'
+        _tmp_args['_local_rsync_password'] = task_vars.get('ansible_ssh_pass') or task_vars.get('ansible_password')
```

Comment:
@TODO: i think here should be just `_tmp_args['_local_rsync_password'] = task_vars.get('ansible_password')`

i think i wrote original line keeping something wrong in mind :D

Reply:
please disregard the comment above -- i've checked again, it seems to be fine